Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/openSAML/SAMLAssertionBuilderException.java

```diff
@@ -0,0 +1,9 @@
+package gov.hhs.fha.nhinc.callback.openSAML;
+
+public class SAMLAssertionBuilderException extends Exception {
+
+    public SAMLAssertionBuilderException(String message)
+    {
```

Comment:
Stylistic suggestion: CONNECT generally does not have open curly braces on their own lines.